apps/codeforafrica/src/utils/camelcaseKeys.js

```diff
@@ -0,0 +1,9 @@
+import { isArray, isObject, transform, camelCase } from "lodash";
+
+export default function convertToCamelCase(obj) {
+  return transform(obj, (acc, value, key, target) => {
+    const camelKey = isArray(target) ? key : camelCase(key);
+
+    acc[camelKey] = isObject(value) ? convertToCamelCase(value) : value;
+  });
+}
```

Comment:
I posted a fix for ESM on Slack. We shouldn't need to import lodash just for this.
>Just had a look at the camelcase-keys / Jest issue Kelvin and the issue seem to be camelcase-keys is built as ESM. You just need to tell Jess to transform it by adding something like :point_down: to `jest.config.js`:
>
> ```js
>   transformIgnorePatterns: ["<rootDir>/node_modules/(?!camelcase-keys)"],
> ```